wrangler-core/src/main/java/io/cdap/directives/row/RecordConditionFilter.java

```diff
@@ -91,19 +91,7 @@ public void destroy() {
     List<Row> results = new ArrayList<>();
     for (Row row : rows) {
       // Move the fields from the row into the context.
-      ELContext ctx = new ELContext(context);
-      for (String var : el.variables()) {
-        Object value = row.getValue(var);
-        String strValue;
-        // support numeric values by converting them to string
-        if (value instanceof Number) {
-          Number number = (Number) value;
```

Comment:
Looks like this number conversion will no longer happen.  I don't know why numbers had to be converted to strings to begin with, but I imagine this change will break existing pipelines.

Reply:
is there test coverage for number input to this directive?

Reply:
Was not there, There was no way to convert something to an number, so this path never happened. So, removed it.